setup-files/gerald-push.yml

```diff
@@ -0,0 +1,21 @@
+name: Notify and Request Reviewers On Push
+'on':
+  push:
+    branches:
+      - master
+      - develop
+      - main
+jobs:
+  gerald:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Same comment for the mobile repo - this is going to be super expensive to run for `webapp`, for example. If we are only reading the latest commit then we should probably remove this!